htdocs/modulebuilder/admin/setup.php

```diff
@@ -137,59 +137,59 @@
 print '<tr class="oddeven">';
 print '<td>'.$langs->trans("UseSpecificEditorName").'</td>';
 print '<td>';
-print '<input type="text" name="MODULEBUILDER_SPECIFIC_EDITOR_NAME" value="'.$conf->global->MODULEBUILDER_SPECIFIC_EDITOR_NAME.'">';
+print '<input type="text" name="MODULEBUILDER_SPECIFIC_EDITOR_NAME" value="'.(isset($conf->global->MODULEBUILDER_SPECIFIC_EDITOR_NAME) ? $conf->global->MODULEBUILDER_SPECIFIC_EDITOR_NAME : "").'">';
```

Comment:
This fix the warning. Great. This is the solution that is used at different places.
This is however the good direction for a better solution:
Use 
`...value="'.getDolGlobalString('MODULEBUILDER_SPECIFIC_EDITOR_NAME').'" ...`

Reply:
I apologize for not having noticed that, even though it was mentioned in the documentation.
I've made the suggested changes using `getDolGlobalString()`  and have repushed the PR. Thank you!